tests/conftest.py

```diff
@@ -64,11 +64,12 @@ def ocp_target_version(request):
 
 
 @pytest.fixture(scope="session")
-def rosa_regions(rosa_allowed_commands):
+def rosa_regions(rosa_allowed_commands, ocm_token):
```

Comment:
do you have a fixture called `ocm_token`?
Also rosa can get ocm client, I think it's better to send it and not the token

Reply:
I can pass client, but [there is a fixture](https://github.com/RedHatQE/managed-services-integration-tests/blob/main/tests/conftest.py#L51) that pass the `ocm_token`, so we have it anyway.